### PR TITLE
[1LP][RFR][NOTEST] Add tests for resolved bugs

### PR DESCRIPTION
**Pull request by jaryn**
Just another bit to decrease the count of BZs with `qa_tests_coverage?` flag.

### CODE REVIEW

File: cfme/tests/cli/test_appliance_update.py [hunk 1]
```diff
@@ -137,6 +137,27 @@ def test_update_yum(appliance_preupdate, appliance):
...
+        testSteps:
+            1. Boot preupdate CFME
+            2. Add repos to update to.
+            3. Do the yum update, check there are no errors in the stderr and
+               perhaps also in stdout.
+
+    Bugzilla:
+        1674055
+    """
+
```
Comment (john-dupuy) [inline cfme/tests/cli/test_appliance_update.py]: Though not functionally necessary, we always have `pass` in manual case definitions

File: cfme/tests/test_db_migrate.py [hunk 2]
```diff
@@ -210,3 +213,34 @@ def test_upgrade_single_inplace(appliance_preupdate, appliance):
...
+        assignee: jhenner
+        casecomponent: Appliance
+        caseimportance: critical
+        initialEstimate: 1/3h
+        startsin: 5.11
+        testSteps:
+            1. Get CFME VMs of preupdate and target version.
+            2. Make sure the ansible is enabled and create an ansible playbook
+               service (for testing the 1735114).
+            2. Turn off the evmserverd processes on both.
```
Comment (john-dupuy) [inline cfme/tests/test_db_migrate.py]: Small typo, I think this should be step 3.

File: cfme/tests/test_db_migrate.py [hunk 3]
```diff
@@ -210,3 +213,34 @@ def test_upgrade_single_inplace(appliance_preupdate, appliance):
...
+    Note this test is quite similar test_db_migrate and perhaps can be removed
+    after the zone checking and ansible checking is implemented there.
+
+    Polarion:
+        assignee: jhenner
+        casecomponent: Appliance
+        caseimportance: critical
+        initialEstimate: 1/3h
+        startsin: 5.11
+        testSteps:
```
Comment (john-dupuy) [inline cfme/tests/test_db_migrate.py]: Can you add `expectedResults`

File: cfme/tests/cli/test_appliance_update.py [hunk 4]
```diff
@@ -137,6 +137,27 @@ def test_update_yum(appliance_preupdate, appliance):
...
+@pytest.mark.meta(coverage=[1674055])
+def test_update_yum_no_errors(old_version):
+    """ Test that the yum update doesn't spill any erros or warnings.
+
+    Polarion:
+        assignee: jhenner
+        caseimportance: high
+        casecomponent: Appliance
+        initialEstimate: 1/4h
+        testSteps:
```
Comment (john-dupuy) [inline cfme/tests/cli/test_appliance_update.py]: Can you add `expectedResults`?

File: cfme/tests/cli/test_appliance_update.py [hunk 5]
```diff
@@ -137,6 +137,27 @@ def test_update_yum(appliance_preupdate, appliance):
     assert result.output in appliance.version
 
 
+@pytest.mark.manual()
```
Comment (john-dupuy) [inline cfme/tests/cli/test_appliance_update.py]: `()` not necessary

File: cfme/tests/cli/test_appliance_update.py (hunk 1 above)
Comment (jaryn) [reply cfme/tests/cli/test_appliance_update.py]: done

File: cfme/tests/test_db_migrate.py (hunk 3 above)
Comment (jaryn) [reply cfme/tests/test_db_migrate.py]: done

File: cfme/tests/cli/test_appliance_update.py (hunk 4 above)
Comment (jaryn) [reply cfme/tests/cli/test_appliance_update.py]: done

File: cfme/tests/test_db_migrate.py [hunk 6]
```diff
@@ -210,3 +213,42 @@ def test_upgrade_single_inplace(appliance_preupdate, appliance):
...
+            7. Check that the service provisioning tab doesn't
+        expectedResults:
+            1.
+            2.
+            3.
+            4.
+            5.
+            6. Zone exists on the target appliance.
+            7. An empty section or a friendly message such as "No Output available"
+    """
```
Comment (ganeshhubale) [inline cfme/tests/test_db_migrate.py]: Missing **pass**

Comment (jaryn) [reply cfme/tests/test_db_migrate.py]: done

File: cfme/tests/configure/test_config_manual.py [hunk 7]
```diff
@@ -336,3 +336,28 @@ def test_subscription_region_unavailable():
...
+    Polarion:
+        assignee: jhenner
+        casecomponent: Configuration
+        caseimportance: high
+        initialEstimate: 1/4h
+        testSteps:
+            1. Settings > Advanced
+            2. Change web_services worker to 1.5.gigabytes or 2.gigabytes
+
+        expectedResults:
```
[the remark this answers is not in the document]
Comment (jaryn) [reply cfme/tests/configure/test_config_manual.py]: done

File: cfme/tests/cli/test_appliance_update.py [hunk 8]
```diff
@@ -137,6 +137,32 @@ def test_update_yum(appliance_preupdate, appliance):
...
+        testSteps:
+            1. Boot preupdate CFME
+            2. Add repos to update to.
+            3. Do the yum update, check there are no errors in the stderr and
+               perhaps also in stdout.
+        expectedResults:
+            1.
+            2.
+            3. no erros in the stderr and stout, yum exit status is 0
+
```
[the remark this answers is not in the document]
Comment (jaryn) [reply cfme/tests/cli/test_appliance_update.py]: done

**Pull request merged**